web/js/index.js

```diff
@@ -50,16 +52,22 @@ const initThreeJsElements = function() {
     controls.noZoom = false;
     controls.noPan = false;
 
-    const $viewport = document.getElementById('gltf-viewport');
     $viewport.appendChild(renderer.domElement);
-
-    window.addEventListener('resize', () => {
-        camera.aspect = window.innerWidth / window.innerHeight;
+    
+    /**
+     * Handles resizing the window.
+     */
+    const handleResize = () => {
+        const width = window.innerWidth,
+            height = (window.innerHeight - document.getElementById('elem-selector').offsetHeight);
```

Comment:
Looks like this is code duplication from line 37

Reply:
Fixed.